store/localcachelayer/file_info_layer.go

```diff
@@ -67,3 +67,33 @@ func (s LocalCacheFileInfoStore) InvalidateFileInfosForPostCache(postId string,
 		s.rootStore.metrics.IncrementMemCacheInvalidationCounter("File Info Cache - Remove by PostId")
 	}
 }
+
+func (s LocalCacheFileInfoStore) GetStorageUsage(allowFromCache, includeDeleted bool) (int64, error) {
```

Comment:
Missing test.

Reply:
I left it out because I don't think it adds anything valuable. The test would essentially be testing the caching logic.